xml/gl.xml

```diff
@@ -6959,7 +6963,7 @@ typedef unsigned int GLhandleARB;
      file) File requests in the Khronos Bugzilla, OpenGL project, Registry
      component. -->
 
-    <enums namespace="GL" start="0x96F0" end="99999" vendor="ARB" comment="RESERVED FOR FUTURE ALLOCATIONS BY KHRONOS">
+    <enums namespace="GL" start="0x9700" end="99999" vendor="ARB" comment="RESERVED FOR FUTURE ALLOCATIONS BY KHRONOS">
         <unused start="0x96F0" end="99999" comment="RESERVED"/>
```

Comment:
This line should likely also be updated to `0x9700` instead of `0x96F0`.

Reply:
Indeed it should. Thanks :)